app/templates/_package.json

```diff
@@ -47,7 +47,8 @@
     "karma-webpack": "2.0.5",<% } else if(testingFramework === 'jest') {%>
     <% if(ngVersionMin == 2) {%>"@types/jest": "16.0.7",<%} 
     else {%>"@types/jest": "21.1.9",<% } %>
-    "jest": "22.0.4",
+    "jest": "22.1.1",
+    "jest-cli": "22.1.1",
```

Comment:
> I found that that jest-cli package not included in the devDependencies. Sure it can be install globally but i think better add as a devDependencies since it required for Jest testing.

This is not a necessary for me, because `jest-cli`is a [`dependencies`](https://github.com/facebook/jest/blob/master/packages/jest/package.json#L6) of `jest`, that means it get automatically installed along with `jest`.

Reply:
Yeah, confirm this. I think maybe `npm install` not finish successfully for that project due to network or something. Anyway I will remove back that dependency in next commit.